diff --git a/dist/Toast.svelte b/dist/Toast.svelte
index b039eebc10a663689dc5cd859c8df815ca86b9a1..d3178ecc67cf4684e7ae4f9d86345726312ca77b 100644
--- a/dist/Toast.svelte
+++ b/dist/Toast.svelte
@@ -315,12 +315,7 @@ const icon = $derived.by(() => {
 
 {#snippet LoadingIcon()}
 	{#if loadingIcon}
-		<div
-			class={cn(classes?.loader, toast?.classes?.loader, 'sonner-loader')}
-			data-visible={toastType === 'loading'}
-		>
-			{@render loadingIcon()}
-		</div>
+		{@render loadingIcon()}
 	{:else}
 		<Loader
 			class={cn(classes?.loader, toast.classes?.loader)}
